config/locales/en-PACIFIC.yml

```diff
@@ -33,8 +33,10 @@ en-PACIFIC:
         irb_status: IRB Status
         irb_number: IRB Number
         version_number: Version
+        official_link: DOI
```

Comment:
@BertZZ Can you add a default translation in `en.yml` to match what was removed from `_attribute_rows.html.erb`?

Reply:
:+1: